CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Bug where an empty destination list in a device's backup set broke creation of DeviceSettings objects for that device.
 
+### Changed
```

Comment:
IT was actually still managing to use the v2 one before, so this does not need a CHANGELOG entry

Reply:
removed.